### ISSUE TITLE
Add support for Thank you/like system

### ISSUE THREAD

**Issue by Eldenroot** [Sama34/OUGC-Awards#35]
As a one of developers of Thank you/like system which is quite popular (over 15000+ downloads on MyBB extend site) I would like to ask you if is possible to add a support for Thank you/like system into requirements for automatic badge (award) reward to users (count of thanks/likes received/given). Thank you for your input

**Comment by Sama34** [comment]
God, I had to check your stats, and indeed 15k downloads is to be praised.

Yes, I will add support for it.

**Comment by Eldenroot** [comment]
Yeah, it is a nice number :) Thank you/like plugin already supports group promotion based on given/received amount of thanks/likes in MyBB but would be really great to extend this for OUGC Awards too :) Thank you!

**Comment by Eldenroot** [comment]
Not related to this issue, just a question - I see that all issues were moved to 1.9 milestone- Does it mean that we will not get any update for MyBB 1.8.x branch? I would like to see one more bigger update - bug fix + few enhancements :) keep good work!

**Comment by Sama34** [comment]
Hi Eldenroot, yeas, that is pretty much what it means.

However, I will still do minor updates for bugs and to include the spanish package, so there is a possibility of one or two new features to be included if enough requests are sent. I would like to include add all features for 1.8 but honestly I don't have enough time so I will need to be selective on my development.

For example, I see this feature and #23  being more relevant in 1.8 since they might not be ported to MyBB 1.9, etc.

If you want to be sure by pushing a PR I will gladly consider accepting it.
